README.md

```diff
@@ -24,12 +25,28 @@ Driver       | Version | patch object | map file   | file size
 ASPI7DOS.SYS | 1.42    | a7-142.obj   | a7-142.map | 36160 bytes
 
 Note that the latest 274x BIOS (2.11) seems to not be available for download from the Adaptec / MicroSemi. Their page has version 2.10. This patch *does not apply to version 2.10*. You can obtain the ROM image by reading it from an controller that has the latest version. The ROM chip is an OTP chip labelled `549306-00 D BIOS 7D00 (c) 1993`.
+The same applies to 284x - only BIOS version 1.01 is available on Adaptec site while controller may have version 2.0. Please note that BIOS version "downgrade" (even non-patched one) can cause problems in controller behavior and use the same or newer BIOS versions than you currently have.
```

Comment:
I don't expect this applies to downgrades only. I expect the hardware to be different between controllers requiring the 1.x and the 2.x BIOS. Incidentally, I got the BIOS v1.01 from an AHA-2842VL (the early revision with a lot of dedicated TTL chips and PALs), whereas your 2.x BIOS was dumped from an AHA-2842A (the late revision that integrated the "bus interface logic" into a single ASIC). So I am confident that we should expand the table to mention "VL" and "A" in the controller column, and also focus on the A/VL difference at this point instead of talking about up/downgrading.